crates/wit-parser/src/ast.rs

```diff
@@ -228,6 +255,45 @@ impl<'a> ExternKind<'a> {
     }
 }
 
+#[derive(Debug)]
+pub struct Include<'a> {
+    /// The path to the world to include
+    from: UsePath<'a>,
+
+    /// The names of the imports and exports to be replaced with
+    names: Vec<UseName<'a>>,
```

Comment:
I think that this isn't quite exactly `UseName` since the `as` keyword is required? For example `with { a, b, c }` doesn't mean much and probably shouldn't parse